cmd/server/cadence/cadence.go

```diff
@@ -82,9 +82,6 @@ func startHandler(c *cli.Context) {
 	sigc := make(chan os.Signal, 1)
 	signal.Notify(sigc, syscall.SIGTERM)
 	for _, svc := range services {
-		if _, ok := cfg.Services[svc]; !ok {
```

Comment:
No need to check here. This is checked down the line in `server.startService` when config is actually needed.